pages/bet_history.html

```diff
@@ -23,13 +23,15 @@
                        {{hist_data.game_date}}
                     </span>
                 </div>
-                <div class = "sel3">Result: {{hist_data.bet_status}}</div>
+                <div class="sel3">Result: {{hist_data.bet_status}}</div>
                 <div><img class='home_img' ng-src="{{hist_data.away_logo}}">{{hist_data.away_team}} <span
                         ng-if="bet_data.final_score_a>-1" class="final_score">{{bet_data.final_score_a}}</span></div>
                 <div><img class='home_img' ng-src="{{hist_data.home_logo}}">{{hist_data.home_team}} <span
                         ng-if="hist_data.final_score_h>-1" class="final_score">{{bet_data.final_score_h}}</span></div>
-                <div>Bet ${{hist_data.amount}} on: {{hist_data.side}} {{hist_data.bet_name}} (<span
-                        ng-if="hist_data.odds>0">+</span>{{hist_data.odds}})
+                <div>Bet ${{hist_data.amount}} on: {{hist_data.side}} <span ng-if="hist_data.bet_name!='moneyline'"><span
```

Comment:
I'm not a fan of it saying moneyline when the bet type is moneyline. Like I said earlier, I think that these bets just say the side and then "to win", but I am fine with leaving it if that is what the group decides. I think that it does add clarity for the casual gambler.